cmd/nerdctl/create/create.go

```diff
@@ -14,16 +14,20 @@
    limitations under the License.
 */
 
-package main
+package create
```

Comment:
Should this be in the `nerdctl/container` package, or maybe better in the `nerdctl/container/create` pkg?

Same applies to the `run`, `exec`, `cp` commands, etc.